exercises/practice/knapsack/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@exercism/javascript-knapsack",
+  "description": "Exercism exercises in Javascript.",
+  "author": "lpizzinidev",
```

Comment:
Not a necessity, but you may want to register (for free) on npmjs: https://www.npmjs.com/~lpizzinidev, so people can find you :)

Reply:
Thanks for the suggestion, I will register 👍🏻